jgrapht-core/src/main/java/org/jgrapht/Graphs.java

```diff
@@ -433,6 +435,159 @@
         }
         return list;
     }
+
+
+    /**
+     * Removes the given vertex from the given graph. If the vertex to be
+     * removed has one or more parents, the parents will be connected directly
+     * to the children of the vertex to be removed.
+     *
+     * @param graph graph to be mutated.
+     * @param vertex vertex to be removed from this graph, if present.
+     *
+     * @return true if the graph contained the specified vertex; false
+     *         otherwise.
+     */
+    public static <V, E extends DefaultEdge> boolean removeVertexAndConnectParentsWithChildren(
```

Comment:
You are right. The utility methods do not depend on DefaultEdge in any way. Therefore this restriction is not required. I removed it. See commit 69bc6fca9f7be131c9afcc1b8fedb67916212877.